### ISSUE TITLE
Distinct name and icon

### ISSUE THREAD

**Issue by cassidyjames** [sk22/megalodon#15]
I see that the name has been changed to Mast`a`don instead of Mast`o`don, I presume to help differentiate from the official app. However, I think a full separate brand or at least a more obviously different name would help those of us out who are using the two apps in parallel to test/compare.

Even if it was something like how Google brands Beta/Canary builds of chrome with distinct icons, e.g. you could call it “Mastodon Fork” with a recolored icon.

**Comment by sk22** [comment]
the icon and the primary theme color are pink now!
still not sure about the name tho... something about "Mastodon Fork" bothers me (i somehow don't like how "Fork" is just dangling there - this could be "fixed" by just naming it "Fork" just like firefox named their nightly builds "Nightly" and not "Firefox Nightly", but no, just "Fork" isn't it either)
i would like to keep the name close to "Mastodon" (as that's what it is), and considering i'm just adding a few features to an existing app, i don't think it needs much of an identity on its own anyway (yet?)
people who use this app probably don't use the official app, so i don't think the distinction being subtle would be much of a problem.
i would, however, stick to "Mastodon for Android Fork" as the "full" app name and would have it listed this way on app stores (thinking about submitting it to f-droid, but i'm not there yet and haven't found the time - plus i'm not sure what package id i should use) - for now, at least.

**Comment by sk22** [comment]
for app names, i've received "Mastodos". "dos" as in 2, maybe also "todos" if i wanted to be bold about the fact that this app has some features "todos" (everyone in spanish) are missing in the official app.
i've left it at "Mastadon" for now, though, as i'm not sure what name we'll go with in the end - maybe some completely other name.

feel free to share your thoughts and ideas!

**Comment by sk22** [comment]
Due to unfortunate circumstances, I have to reopen this issue.

**Comment by sk22** [comment]
> How about **Mustard**?

that kind of sounds like an amazing name for an app, but i just happen to dislike mustard

**Comment by sk22** [comment]
someone I know suggested "Pink Elephant" - any opinions?

**Comment by lilylunatic** [comment]
I did a thingy that could be used as an icon - I'll can provide the xcf for further refinements/changes ![ic_launcher-playstore](https://user-images.githubusercontent.com/78930853/203299210-fccab3ce-fcad-4d92-85b3-bd808619458e.png)

**Comment by sk22** [comment]
this is perfect

**Comment by sk22** [comment]
i think i'll just replicate this using inkscape or something and then do the refinements from there. thank you!!

**Comment by lilylunatic** [comment]
I'll send you the file via matrix - it's mostly paths anyways

**Comment by cassidyjames** [comment]
I love Pink Elephant, especially if you're going to keep the pink accent around; it feels cute and whimsical while clearly relating to Mastodon. But ultimately whatever you like that is distinct from other apps will be fine. :)

**Comment by jont4** [comment]
What you think about "Elefante"?
It's Elephant in PT

**Comment by Antitoxic0** [comment]
I have a list of suggestions:
- TRUNKY: from "trunk", the long nose of elephant + y like tusk-y (maybe too similar)
- CALFY: from "calf" baby elephant + y. Or simply CALF
- HOOTER or SCHNOZZ: a jokey way to say nose, trunk

**Comment by neilideal** [comment]
> I did a thingy that could be used as an icon - I'll can provide the xcf for further refinements/changes ![ic_launcher-playstore](https://user-images.githubusercontent.com/78930853/203299210-fccab3ce-fcad-4d92-85b3-bd808619458e.png)

It would be fun to play up the word shark in the title. And even better, find some video from National Geographic or Discovery about the ancient underwater world and remember what such sharks were called. I think it was Megalodon, but I'm not quite sure about that.

**Comment by LucasGGamerM** [comment]
I think a nice name would be musty

**Comment by th0mcat** [comment]
Wooly

Mastodon -> Wooly Mammoth -> Wooly

**Comment by LucasGGamerM** [comment]
That's also quite great imo

**Comment by lichingyun** [comment]
Megalodon is a beautiful name 🤩🦈

**Comment by sk22** [comment]
> > I did a thingy that could be used as an icon - I'll can provide the xcf for further refinements/changes ![ic_launcher-playstore](https://user-images.githubusercontent.com/78930853/203299210-fccab3ce-fcad-4d92-85b3-bd808619458e.png)
> 
> It would be fun to play up the word shark in the title. And even better, find some video from National Geographic or Discovery about the ancient underwater world and remember what such sharks were called. I think it was Megalodon, but I'm not quite sure about that.

i think you won, thank you!! https://floss.social/@mastodos/109398610778057798

**Comment by Choukajohn** [comment]
What do you think about the extinct cousin of Mastodons,
Stegodon ?

**Comment by cassidyjames** [comment]
I also love the sound of Megalodon, and it sounds like a mega-version of Mastodon. 😁 Which, with more features, makes sense!

**Comment by LucasGGamerM** [comment]
Hey guys, i think i have something of a home timeline header here 
![image](https://user-images.githubusercontent.com/71328265/205657738-2e0fee1a-4c8c-4f4d-a003-13bc759c978c.png)
I'm putting it inside the app to test it out, will send screenshots soon

**Comment by sk22** [comment]
ooohhh this looks neat!

**Comment by LucasGGamerM** [comment]
Lets just file another pull request just in case

**Comment by LucasGGamerM** [comment]
![megalodon3](https://user-images.githubusercontent.com/71328265/205708023-f6060af9-84f3-426f-bbe7-bb5723315ec5.svg)
Here is the file in case anyone is interested
